files/en-us/learn/javascript/building_blocks/events/index.html

```diff
@@ -524,26 +524,22 @@ <h4 id="Bubbling_and_capturing_explained">Bubbling and capturing explained</h4>
 
 <ul>
  <li>The browser checks to see if the element's outer-most ancestor ({{htmlelement("html")}}) has an <code>onclick</code> event handler registered on it for the capturing phase, and runs it if so.</li>
- <li>Then it moves on to the next element inside <code>&lt;html&gt;</code> and does the same thing, then the next one, and so on until it reaches the element that was actually selected.</li>
+ <li>Then it moves on to the next element inside <code>&lt;html&gt;</code> and does the same thing, then the next one, and so on until it reaches the direct parent of the element that was actually selected.</li>
 </ul>
 
 <p>In the <strong>target</strong> phase:</p>
 <ul>
- <li>The browser calls any event listeners registered for this phase (which is the point at which the event has arrived at {{domxref("EventTarget", "the event's target", "", 1)}}).</li>
- <li>Then, if {{domxref("Event.bubbles")}} is <code>true</code>, it propagates the event to the next immediate ancestor element, then the next one, and so on until it reaches the <code>&lt;html&gt;</code> element. Otherwise, if {{domxref("Event.bubbles")}} is <code>false</code>, it doesn’t propagate the event to any ancestors of the target.</li>
+ <li>The browser checks to see if the {{domxref("Event.target", "target")}} property has an <code>onclick</code> event handler registered on it, and runs it if so.</li>
```

Comment:
```suggestion
 <li>The browser checks to see if the {{domxref("Event.target", "target")}} property has an event handler for the <code>click</code> event registered on it, and runs it if so.</li>
```

What about this? That way we don't assume how the event handler is registered.

Reply:
> What about this? That way we don't assume how the event handler is registered.

Yup, agreed